tsconfig.json

```diff
@@ -1,26 +1,30 @@
+/* To learn more about this file see: https://angular.io/config/tsconfig. */
 {
   "compileOnSave": false,
   "compilerOptions": {
-    "baseUrl": "./",
-    "downlevelIteration": true,
-    "importHelpers": true,
     "outDir": "./dist/out-tsc",
+    "forceConsistentCasingInFileNames": true,
+    "strict": true,
+    "noImplicitOverride": true,
+    "noPropertyAccessFromIndexSignature": true,
+    "noImplicitReturns": true,
+    "noFallthroughCasesInSwitch": true,
+    "skipLibCheck": true,
+    "esModuleInterop": true,
```

Comment:
non-blocking

Is this default now for Angular or is this a decision to turn on? I assume this is why the import style changed for socket.io

Reply:
Yes it is the default. It is because of esbuild. I created an app with ng new and then brought that config to this repo.